sagemaker-training-compiler/tensorflow/single_gpu_single_node/hyper-parameter-tuning.ipynb

```diff
@@ -0,0 +1,718 @@
+{
```

Comment:
<p>Grammar nit - no space before exclamation point - 2 occurrences in the notebook. ie <code>40 !</code> -&gt; <code>40!</code>   </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/3475/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='18'/>